### PR TITLE
DOC:Oudated flags for print_qiime_config.py in doc

### PR DESCRIPTION
**Pull request by ElDeveloper**
The tutorial suggested using print_qiime_config.py -t, which no longer
outputs the text that shows whether sourcetracker is installed or not.
To see this text you need to pass `-tfv`.

### CODE REVIEW

**Comment by ElDeveloper** [comment]
I also noticed a few other things that were out of date.

**Comment by qiime-r2d2** [comment]
Build results will soon be (or already are) available at: http://ci.qiime.org/job/qiime-github-pr/1657/

File: doc/tutorials/source_tracking.rst [hunk 1]
```diff
@@ -20,7 +20,7 @@ Test Usage
 ----------
 Before running SourceTracker check to see if it installed and accessible to QIIME::
 
-    print_qiime_config.py -t
+    print_qiime_config.py -tfv
```
Comment (gregcaporaso) [inline doc/tutorials/source_tracking.rst]: What does the ``v`` do here? I think just ``-tf`` gets it.

Comment (ElDeveloper) [inline doc/tutorials/source_tracking.rst]: The specific message that the documentation suggests you look for is
`sourcetracker is installed ... ok` which only comes up if you add the
-v flag.

Comment (gregcaporaso) [inline doc/tutorials/source_tracking.rst]: Got it, thanks!

**Pull request merged**